src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -285,7 +270,7 @@ export class DotEditContentHtmlService {
             const target = <HTMLElement>$event.target;
             const container = <HTMLElement>target.closest('div[data-dot-object="container"]');
 
-            this.contentletEvents.next({
+            this.iframeActions.next({
```

Comment:
We were doing handling events from the jsp and events from the toolbars we inject to the iframe with the same Subject, not good... so I create another Subject just for the toolbar actions.